conanfile.py

```diff
@@ -19,7 +19,7 @@ class ConanFileDefault(ConanFileBase):
 
     def requirements(self):
         if self.options.with_openssl:
-            self.requires("openssl/1.1.1d")
+            self.requires("OpenSSL/1.1.1c@conan/stable")
```

Comment:
Why?

Reply:
it looks like `openssl/1.1.1d@conan/stable` isn't available anymore on conan center, that's why I stepped back to 1.1.1c

Reply:
Use from Conan Center Index. It's the official now and OpenSSL form Conan Community is totally deprecated.

Reply:
still, 1.1.1c:stable is the latest available